packages/documentation/src/stories/components/checkbox/checkbox.stories.tsx

```diff
@@ -0,0 +1,304 @@
+      }
+    },
+    invalidFeedback: {
+      name: 'Invalid Feedback',
+      description: 'Text to show when the component is in an invalid state.',
+      control: {
+        type: 'text'
+      },
+      table: {
+        category: 'States'
+      }
+    }
+  }
+} as Meta;
+
+function toggle (args: Args, updateArgs: Function) {
+  updateArgs({ indeterminate: false });
+  updateArgs({ checked: !args.checked });
+}
+
+const Template = (args: Args, story: Story) => {
+  const [_, updateArgs] = useArgs();
+
+  const id = `ExampleCheckbox_${story.name}`;
+  const classes = [
+    'form-check-input',
+    args.validation
+  ].filter(c => c && c !== 'null').join(' ');
+  
+  const contextuals: (JSX.Element | null)[] = [
+    args.validation === 'is-valid' ? <p key="valid" className="valid-feedback">{ args.validFeedback }</p> : null,
+    args.validation === 'is-invalid' ? <p key="invalid" className="invalid-feedback">{ args.invalidFeedback }</p> : null
+  ].filter(f => f !== null);
+
+  setTimeout(function () {
+    const input: HTMLInputElement | null = document.querySelector('input.form-check-input');
+    if (input) input.indeterminate = args.indeterminate;
+  }, 0);
+
+  return <div className="form-check">
+    <input
+      id={ id }
+      className={ classes }
+      type="checkbox"
+      value={ args.value }
+      checked={ args.checked }
+      disabled={ args.disabled }
+      aria-label={ args.labelHidden ? args.label : undefined }
+      onChange={ (e: React.ChangeEvent) => toggle(args, updateArgs) }
+    />
+    { !args.labelHidden ? <label htmlFor={ id } className="form-check-label">{ args.label }</label> : null }
+    { contextuals }
+  </div>;
+};
+
+export const Default = Template.bind({});
+Default.decorators = [
+  (Story: Story, { args }) => <div className={ `p-4 pb-2 ${args.background}` }>
+    <Story/>
+  </div>
+];
+
+const TemplateInline = (args: Args) => [
+  <div key="FormCheck_1" className="form-check form-check-inline">
+    <input id="ExampleCheckbox_Inline_1" className="form-check-input" type="checkbox"/>
+    <label htmlFor="ExampleCheckbox_Inline_1" className="form-check-label">{ args.label }</label>
+  </div>,
+  <div key="FormCheck_2" className="form-check form-check-inline">
+    <input id="ExampleCheckbox_Inline_2" className="form-check-input" type="checkbox"/>
+    <label htmlFor="ExampleCheckbox_Inline_2" className="form-check-label">{ args.label }</label>
+  </div>,
+  <div key="FormCheck_3" className="form-check form-check-inline">
+    <input id="ExampleCheckbox_Inline_3" className="form-check-input" type="checkbox"/>
+    <label htmlFor="ExampleCheckbox_Inline_3" className="form-check-label">{ args.label }</label>
+  </div>,
+  <div key="FormCheck_4" className="form-check form-check-inline">
+    <input id="ExampleCheckbox_Inline_4" className="form-check-input" type="checkbox"/>
+    <label htmlFor="ExampleCheckbox_Inline_4" className="form-check-label">{ args.label }</label>
+  </div>
```

Comment:
Can you wrap this example in a `<fieldset><legend>...</legend> ...checkboxes... </fieldset>` structure to have a label for the group of checkboxes?

This should probably also be mentioned in the docs (https://www.accessibility-developer-guide.com/examples/forms/grouping-with-fieldset-legend/).